Ticket05/dashboard.js

claramiralles opened the pull request:
Hi @carmen-tm,

I have added the Dashboard.js component with a nav link from App.js and the route path. I have also added the filter by date elements in a DateFilter.js, without styles, only the elements, to reuse as much as we could from the previous code.

Could you please review it?

Thanks a lot!

carmen-tm commented on src/components/Dashboard.js:
```diff
@@ -0,0 +1,26 @@
+    render () {
+        return (
+            <div>
+                <DateFilter 
```
Hi! instead of this `DateFilter` component, I would propose the following architecture:
- Inside Dashboard: `Filters` and `Graphics` components. 
- Inside `Filters`, the `DateFilter` one, plus another one that in the future will be `CompanyFilter` or something like that.
- You can leave `Graphics` as an empty space for the coming two `Chart` components
:)

You can have a look here:
[https://creately.com/app/?diagid=jweysu821#](url)

carmen-tm commented:
Also, another little thing is that we have an error on console related to `DateFilter.js` like the following, but I'm not sure what is about `Warning: validateDOMNesting(...): <th> cannot appear as a child of <div>.`. We can review that later, is not that urgent :)

claramiralles commented:
Hi @carmenpmoreno I have added the component Graphics where later we will include the components Chart. I have also added the component Filters and instance the component DateFilter.js. Here later we will also include the other components for the filters.
Let me know if you need any extra info! Thanks a lot!
